.azure-pipelines/config.yml

```diff
@@ -145,6 +145,35 @@ jobs:
       pathtoPublish: '$(Build.ArtifactStagingDirectory)'
       artifactName: 'bundle'
 
+- job: 'win_docker_build'
+  pool:
+    vmImage: 'windows-2019'
+  steps:
+  - template: 'templates/changes-include.yml'
+    parameters:
+      paths: 'go.mod go.sum Dockerfile.windows .azure-pipelines'
+  - bash: |
+      set -euxo pipefail
+
+      if [[ "$BUILD_SOURCEBRANCH" =~ ^refs/tags/v\d+\.\d+\.\d+$ ]]; then
```

Comment:
If we are pushing final releases, this job should be limited to only tag pushes.

Reply:
So don't build for PRs if there are changes to Dockerfile.windows and other related files?

Reply:
We should only push to the `signalfx/signalfx-agent` quay repo for production releases, and it has to build it every time because the Windows docker image includes the agent code, not just stuff in DOckerfile.windows/go.mod, etc.